cmd/tailscale/cli/update.go

```diff
@@ -144,7 +144,19 @@ func newUpdater() (*updater, error) {
 		case distro.Debian: // includes Ubuntu
 			up.update = up.updateDebLike
 		case distro.Arch:
-			up.update = up.updateArch
+			up.update = up.updateArchLike
+		}
+		switch {
+		case haveExecutable("pacman"):
+			up.update = up.updateArchLike
+		case haveExecutable("apt-get"):
```

Comment:
Left a TODO, since `updateDebLike` assumes `apt-get` and will fail if that's missing, even if `apt` is present.
Will add `apt` support in a separate PR.